compiler/src/iree/compiler/Dialect/Flow/Transforms/test/outline_dispatch_regions.mlir

```diff
@@ -179,11 +179,39 @@ func.func @dispatchWithCountRegion(%arg0: tensor<4xi32>) -> tensor<4xi32> {
 
 // -----
 
+//      CHECK: flow.executable private @main_dispatch_0 {
+// CHECK-NEXT:   flow.executable.export public @main_dispatch_0_generic_4x8_i32xf32
+//      CHECK: func.func @main_dispatch_0_generic_4x8_i32xf32(
+func.func @main() -> tensor<4x8xf32> {
```

Comment:
Can you either add a comment about what the test case is covering or pick a test case name for this instead of "main" (`dispatchWithDifferentTypes`?). Looks like the other test cases I added just used `main` with comments :P